### ISSUE TITLE
Simplify.

### ISSUE THREAD

**Issue by kalverra** [rejkpp/colmaracademy#3]
https://github.com/rejkpp/colmaracademy/blob/gh-pages/index.html#L180-L188

This is a cool use of jQuery, and if it fits your design, go right ahead and use it. But there's also a simpler way just through HTML you can use here: https://github.com/rejkpp/colmaracademy/blob/gh-pages/index.html#L180-L188

**Comment by rejkpp** [comment]
Yes, I just wanted a play button, without the rest of the controls.
